service/provider/src/main/java/gov/va/vro/service/provider/mas/service/MasApiService.java

```diff
@@ -87,18 +89,33 @@ public String orderExam(MasOrderExamReq masOrderExamReq) throws MasException {
     try {
       String url = masApiProps.getBaseURL() + masApiProps.getCreateExamOrderPath();
       HttpHeaders headers = getMasHttpHeaders();
+      ObjectMapper mapper = new ObjectMapper();
+
+      try {
+        // convert user object to json string and return it
+        log.info("masOrderExamReq JSON : " + mapper.writeValueAsString(masOrderExamReq));
```

Comment:
we could just use toString() here with Lombok @ToStrring() to avoid the Json mapper